resources/aws_ec2.go

```diff
@@ -374,19 +383,28 @@ func (obj *AwsEc2Res) Watch() error {
 	if obj.WatchListenAddr != "" {
 		return obj.snsWatch()
 	}
-	return obj.longpollWatch()
+	return obj.waiterWatch()
 }
 
-// longpollWatch uses the ec2 api's built in methods to watch ec2 resource state.
-func (obj *AwsEc2Res) longpollWatch() error {
-	send := false
+// waiterWatch uses the ec2 api's built in methods to watch ec2 resource state.
+func (obj *AwsEc2Res) waiterWatch() error {
+	var send bool
 	var exit *error
+
+	// we tell the engine we're running right away. Because watchWaiter
+	// checks the instance state directly so we don't miss any events.
 	if err := obj.Running(); err != nil {
 		return err
 	}
+
+	// canellable context used for exiting cleanly.
```

Comment:
typo. one liners are either full sentences with capitalization and period or have neither same above and in a bunch of other comments.

Reply:
I guess I picked this up as a bad habit somewhere along the way. Will correct all of them.